core/arch/arm/kernel/secure_partition.c

```diff
@@ -68,6 +85,30 @@ struct sp_session *sp_get_session(uint32_t session_id)
 	return NULL;
 }
 
+TEE_Result sp_partition_info_get_all(struct ffa_partition_info *fpi,
+				     size_t *elem_count)
+{
+	size_t in_count = *elem_count;
+	struct sp_session *s = NULL;
+	size_t count = 0;
+
+	TAILQ_FOREACH(s, &open_sp_sessions, link) {
+		if (s->state != sp_dead)
```

Comment:
Back on this and https://github.com/OP-TEE/optee_os/pull/4556#pullrequestreview-655451731: this implementation returns info of dead SP sessions only. Sounds strange.

Reply:
Done